### ISSUE TITLE
lb: backend server connection cannot be restored instantly after closing

### ISSUE THREAD

**Issue by vdmit11** [natsys/tempesta#63]
NOTE:
-  The master branch doesn't have the feature and the bug can't be reproduced there.
    Don't touch it until the `failover` branch is merged.

Scenario:
1. start Tempesta FW
2. start a backend server (e.g. `service nginx start`)
3. wait until the connection is established
4. restart the backend server

Expected:
The connection should be re-established (may be checked via `netstat` and a corresponding log message appears).

Got:
The connection is not re-established.
When the backend server is stopped (but not yet started) there is a log message:

    IPv4: Attempt to release TCP socket in state 1
    [sync_socket] process error on socket %p

Basic investigation shown the following scenario of how that happens:
1. The connection is closed, and Tempesta catches that and tries to re-connect to the server (which is offline now).
2. `kernel_connect()` actually succeeds and returns an alive socked without any error (but should fail because the server is offline and no connection is possible).
3. Tempesta takes the socket and thinks that the connection is re-established, so the background thread that establishes connections wents into sleep until a disconnect occurs.
4. The socket returned by `kernel_connect()` dies shortly after (asynchronously), but the error happens during handling this event (the ` IPv4: Attempt to release TCP socket in state 1` message).
5. No disconnect callbacks invoked (because of the error) and Tempesta doesn't detect the disconnect and doesn't even try to re-establish the connection. The socket seems to be not released properly.

If you add a delay (i.e. `msleep()`) between catching the disconnect event and `kernel_connect()` then the problem doesn't occur (the `kernel_connect()` returns an error as it should).

**Comment by keshonok** [comment]
Part of this issue was fixed in c160784

**Comment by keshonok** [comment]
The issue can not be reproduced after a series of Tempesta code changes. Reconnects are handled in a different manner now, and everything works as expected. When a back-end server is restarted Tempesta catches that and starts the reconnect procedure. It is repeated until new connection is established with the server. New socket is created for each reconnect attempt, and old socket is properly closed. Please see the trace below.

```
Loading Tempesta kernel modules...
[   65.887163] [tdb] Start Tempesta DB
[   66.046534] [tempesta] Initializing Tempesta FW kernel module...
[   66.047310] [tempesta] init: cfg_if
[   66.047757] [tempesta] init: tls
[   66.048203] [tempesta] init: http
[   66.048638] [tempesta] init: http_sticky
[   66.051296] [tempesta] init: server
[   66.051782] [tempesta] init: client
[   66.052259] [tempesta] init: sock_srv
[   66.052744] [tempesta] init: sock_clnt
[   66.053233] [tempesta]   register cfg: filter
[   66.053779] [tempesta]   register cfg: cache
[   66.054334] [tempesta]   register cfg: http_sticky
[   66.054946] [tempesta]   register cfg: sock_srv
[   66.055527] [tempesta]   register cfg: sock_clnt
[   66.070460] [tempesta] Registering new scheduler: dummy
[   66.087327] [tempesta]   tfw_sched_hash: init
[   66.087745] [tempesta] Registering new scheduler: hash
[   66.103561] [tempesta]   tfw_sched_http: init
[   66.103983] [tempesta]   register cfg: tfw_sched_http
[   66.104450] [tempesta] Registering new scheduler: http
[   66.118153] [tempesta]   tfw_sched_rr: init
[   66.118566] [tempesta] Registering new scheduler: round-robin
Starting Tempesta...
[   66.122553] [tempesta] got state via sysctl: start
[   66.123179] [tempesta]   reading configuration file...
[   66.123842] [tempesta]   reading file: /home/natsys-lab/tfw-run/tempesta/etc/tempesta_fw.conf
[   66.125711] [tempesta]   file size: 5783 bytes
[   66.126320] [tempesta]   read by offset: 0
[   66.126921] [tempesta]   read by offset: 4096
[   66.127602] [tempesta]   read by offset: 5783
[   66.128257] [tempesta] starting all modules...
[   66.128832] [tempesta]   parsing configuration and pushing it to modules...
[   66.129711] [tempesta]   spec handle: 'server'
[   66.130338] [tempesta]   new server group: 'default'
[   66.131396] [tempesta]   spec handle: 'cache'
[   66.132371] [tempesta]   use default entry: 'filter_tbl_size 16777216;'
[   66.133395] [tempesta]   spec handle: 'filter_tbl_size'
[   66.134044] [tempesta]   use default entry: 'filter_db /opt/tempesta/db/filter.tdb;'
[   66.135034] [tempesta]   spec handle: 'filter_db'
[   66.135623] [tempesta]   use default entry: 'cache_size 268435456;'
[   66.136427] [tempesta]   spec handle: 'cache_size'
[   66.137033] [tempesta]   use default entry: 'cache_db /opt/tempesta/db/cache.tdb;'
[   66.137963] [tempesta]   spec handle: 'cache_db'
[   66.138556] [tempesta]   use default entry: 'listen 80;'
[   66.139225] [tempesta]   spec handle: 'listen'
[   66.139785] [tempesta]   starting modules...
[   66.140334] [tempesta]   mod_start(): filter
[   66.147131] [tdb] Opened table /opt/tempesta/db/filter.tdb: size=16777216 rec_size=20 base=ffff880033000000
[   66.148059] [tempesta]   mod_start(): cache
[   66.148439] [tempesta]   mod_start(): http_sticky
[   66.148859] [tempesta]   mod_start(): sock_srv
[   66.169768] [tempesta]   mod_start(): sock_clnt
[   66.170298] [sync_sockets]   ss_tcp_state_change: sk ffff880038ca0940, sk->sk_socket           (null), state (Established)
[   66.171352] [tempesta]   connected: 192.168.10.1:8080
[   66.171874] [sync_sockets]   ss_tcp_state_change: sk ffff880039961280, sk->sk_socket           (null), state (Established)
[   66.172987] [tempesta]   connected: 192.168.10.1:8080
[   66.173486] [sync_sockets]   ss_tcp_state_change: sk ffff880039960980, sk->sk_socket           (null), state (Established)
[   66.174523] [tempesta]   connected: 192.168.10.1:8080
[   66.175059] [sync_sockets]   ss_tcp_state_change: sk ffff880039960080, sk->sk_socket           (null), state (Established)
[   66.177037] [tempesta]   connected: 192.168.10.1:8080
[   66.177657] [tempesta] Open listen socket on: 0.0.0.0
[   66.178178] [tempesta]   start listening on socket: sk=ffff8800399e52c0
[   66.178793] [tempesta]   mod_start(): tfw_sched_http
[   66.179253] [tempesta]   spec handle: 'sched_http_rules'
[   66.179742] [tempesta]   tfw_sched_http: begin sched_http_rules
[   66.180298] [tempesta]   spec handle: 'match'
[   66.180707] [tempesta]   tfw_sched_http: parsed rule: match  'default'=ffff880038c6e640  '*'=1  '*'=1  '*'
[   66.181578] [tempesta]   tfw_sched_http: finish sched_http_rules
[   66.182122] [tempesta] modules are started
net.tempesta.state = start
done
done
[   86.553675] [sync_sockets]   ss_tcp_state_change: sk ffff880038ca0940, sk->sk_socket           (null), state (Close Wait)
[   86.555427] [sync_sockets]   Peer connection closing
[   86.556191] [tempesta]   connection lost: 192.168.10.1:8080
[   86.557105] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880038ca0940 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880038ca0940, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   free rcv skb ffff880038a9f1c0
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff880039961280, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   Peer connection closing
[   86.557630] [tempesta]   connection lost: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880039961280 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880039961280, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   free rcv skb ffff880038a9f1c0
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff880039960980, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   Peer connection closing
[   86.557630] [tempesta]   connection lost: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880039960980 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880039960980, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   free rcv skb ffff880038a9f1c0
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff880039960080, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   Peer connection closing
[   86.557630] [tempesta]   connection lost: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880039960080 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880039960080, sk->sk_socket           (null), state (Close Wait)
[   86.557630] [sync_sockets]   free rcv skb ffff880038a9f1c0
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff8800399e40c0, sk->sk_socket           (null), state (Close)
[   86.557630] [tempesta]   connection error: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff8800399e40c0 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff8800399e40c0, sk->sk_socket           (null), state (Close)
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff8800399e49c0, sk->sk_socket           (null), state (Close)
[   86.557630] [tempesta]   connection error: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff8800399e49c0 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff8800399e49c0, sk->sk_socket           (null), state (Close)
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff880038ac1300, sk->sk_socket           (null), state (Close)
[   86.557630] [tempesta]   connection error: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880038ac1300 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880038ac1300, sk->sk_socket           (null), state (Close)
[   86.557630] [sync_sockets]   ss_tcp_state_change: sk ffff880038ac0a00, sk->sk_socket           (null), state (Close)
[   86.557630] [tempesta]   connection error: 192.168.10.1:8080
[   86.557630] [tempesta]   Free msg:           (null)
[   86.557630] [sync_sockets]   Close socket ffff880038ac0a00 (account=1)
[   86.557630] [sync_sockets]   ss_do_close: sk ffff880038ac0a00, sk->sk_socket           (null), state (Close)
[   86.676100] [sync_sockets]   ss_tcp_state_change: sk ffff880038ac0100, sk->sk_socket           (null), state (Close)
[   86.677036] [tempesta]   connection error: 192.168.10.1:8080
[   86.677539] [tempesta]   Free msg:           (null)
[   86.677973] [sync_sockets]   Close socket ffff880038ac0100 (account=1)
[   86.678560] [sync_sockets]   ss_do_close: sk ffff880038ac0100, sk->sk_socket           (null), state (Close)
[   86.679443] [sync_sockets]   ss_tcp_state_change: sk ffff88003854b340, sk->sk_socket           (null), state (Close)
[   86.680091] [tempesta]   connection error: 192.168.10.1:8080
[   86.680091] [tempesta]   Free msg:           (null)
[   86.680091] [sync_sockets]   Close socket ffff88003854b340 (account=1)
[   86.680091] [sync_sockets]   ss_do_close: sk ffff88003854b340, sk->sk_socket           (null), state (Close)
[   86.680091] [sync_sockets]   ss_tcp_state_change: sk ffff88003854aa40, sk->sk_socket           (null), state (Close)
[   86.680091] [tempesta]   connection error: 192.168.10.1:8080
[   86.680091] [tempesta]   Free msg:           (null)
[   86.680091] [sync_sockets]   Close socket ffff88003854aa40 (account=1)
[   86.680091] [sync_sockets]   ss_do_close: sk ffff88003854aa40, sk->sk_socket           (null), state (Close)
[   86.680091] [sync_sockets]   ss_tcp_state_change: sk ffff88003854a140, sk->sk_socket           (null), state (Close)
[   86.680091] [tempesta]   connection error: 192.168.10.1:8080
[   86.680091] [tempesta]   Free msg:           (null)
[   86.680091] [sync_sockets]   Close socket ffff88003854a140 (account=1)
[   86.688413] [sync_sockets]   ss_do_close: sk ffff88003854a140, sk->sk_socket           (null), state (Close)
[   86.689374] [sync_sockets]   ss_tcp_state_change: sk ffff880038773380, sk->sk_socket           (null), state (Close)
[   86.690313] [tempesta]   connection error: 192.168.10.1:8080
[   86.690820] [tempesta]   Free msg:           (null)
[   86.691257] [sync_sockets]   Close socket ffff880038773380 (account=1)
[   86.691842] [sync_sockets]   ss_do_close: sk ffff880038773380, sk->sk_socket           (null), state (Close)
[   86.692736] [sync_sockets]   ss_tcp_state_change: sk ffff880038772a80, sk->sk_socket           (null), state (Close)
[   86.693368] [tempesta]   connection error: 192.168.10.1:8080
[   86.693368] [tempesta]   Free msg:           (null)
[   86.693368] [sync_sockets]   Close socket ffff880038772a80 (account=1)
[   86.693368] [sync_sockets]   ss_do_close: sk ffff880038772a80, sk->sk_socket           (null), state (Close)
[   86.693368] [sync_sockets]   ss_tcp_state_change: sk ffff880038772180, sk->sk_socket           (null), state (Close)
[   86.693368] [tempesta]   connection error: 192.168.10.1:8080
[   86.693368] [tempesta]   Free msg:           (null)
[   86.693368] [sync_sockets]   Close socket ffff880038772180 (account=1)
[   86.693368] [sync_sockets]   ss_do_close: sk ffff880038772180, sk->sk_socket           (null), state (Close)
[   86.693368] [sync_sockets]   ss_tcp_state_change: sk ffff8800388213c0, sk->sk_socket           (null), state (Close)
[   86.693368] [tempesta]   connection error: 192.168.10.1:8080
[   86.693368] [tempesta]   Free msg:           (null)
[   86.693368] [sync_sockets]   Close socket ffff8800388213c0 (account=1)
[   86.693368] [sync_sockets]   ss_do_close: sk ffff8800388213c0, sk->sk_socket           (null), state (Close)
[   86.704116] [sync_sockets]   ss_tcp_state_change: sk ffff880038820ac0, sk->sk_socket           (null), state (Close)
[   86.705026] [tempesta]   connection error: 192.168.10.1:8080
[   86.705509] [tempesta]   Free msg:           (null)
[   86.705926] [sync_sockets]   Close socket ffff880038820ac0 (account=1)
[   86.706479] [sync_sockets]   ss_do_close: sk ffff880038820ac0, sk->sk_socket           (null), state (Close)
[   86.707314] [sync_sockets]   ss_tcp_state_change: sk ffff8800388201c0, sk->sk_socket           (null), state (Close)
[   86.708104] [tempesta]   connection error: 192.168.10.1:8080
[   86.708104] [tempesta]   Free msg:           (null)
[   86.708104] [sync_sockets]   Close socket ffff8800388201c0 (account=1)
[   86.708104] [sync_sockets]   ss_do_close: sk ffff8800388201c0, sk->sk_socket           (null), state (Close)
[   86.708104] [sync_sockets]   ss_tcp_state_change: sk ffff880038865400, sk->sk_socket           (null), state (Close)
[   86.708104] [tempesta]   connection error: 192.168.10.1:8080
[   86.708104] [tempesta]   Free msg:           (null)
[   86.708104] [sync_sockets]   Close socket ffff880038865400 (account=1)
[   86.708104] [sync_sockets]   ss_do_close: sk ffff880038865400, sk->sk_socket           (null), state (Close)
[   86.708104] [sync_sockets]   ss_tcp_state_change: sk ffff880038864b00, sk->sk_socket           (null), state (Close)
[   86.708104] [tempesta]   connection error: 192.168.10.1:8080
[   86.708104] [tempesta]   Free msg:           (null)
[   86.708104] [sync_sockets]   Close socket ffff880038864b00 (account=1)
[   86.708104] [sync_sockets]   ss_do_close: sk ffff880038864b00, sk->sk_socket           (null), state (Close)
[   86.720130] [sync_sockets]   ss_tcp_state_change: sk ffff880038864200, sk->sk_socket           (null), state (Close)
[   86.721053] [tempesta]   connection error: 192.168.10.1:8080
[   86.721537] [tempesta]   Free msg:           (null)
[   86.721957] [sync_sockets]   Close socket ffff880038864200 (account=1)
[   86.722515] [sync_sockets]   ss_do_close: sk ffff880038864200, sk->sk_socket           (null), state (Close)
[   86.723358] [sync_sockets]   ss_tcp_state_change: sk ffff8800388ad440, sk->sk_socket           (null), state (Close)
[   86.724119] [tempesta]   connection error: 192.168.10.1:8080
[   86.724119] [tempesta]   Free msg:           (null)
[   86.724119] [sync_sockets]   Close socket ffff8800388ad440 (account=1)
[   86.724119] [sync_sockets]   ss_do_close: sk ffff8800388ad440, sk->sk_socket           (null), state (Close)
[   86.724119] [sync_sockets]   ss_tcp_state_change: sk ffff8800388acb40, sk->sk_socket           (null), state (Close)
[   86.724119] [tempesta]   connection error: 192.168.10.1:8080
[   86.724119] [tempesta]   Free msg:           (null)
[   86.724119] [sync_sockets]   Close socket ffff8800388acb40 (account=1)
[   86.724119] [sync_sockets]   ss_do_close: sk ffff8800388acb40, sk->sk_socket           (null), state (Close)
[   86.724119] [sync_sockets]   ss_tcp_state_change: sk ffff8800388ac240, sk->sk_socket           (null), state (Close)
[   86.724119] [tempesta]   connection error: 192.168.10.1:8080
[   86.724119] [tempesta]   Free msg:           (null)
[   86.724119] [sync_sockets]   Close socket ffff8800388ac240 (account=1)
[   86.724119] [sync_sockets]   ss_do_close: sk ffff8800388ac240, sk->sk_socket           (null), state (Close)
[   86.736121] [sync_sockets]   ss_tcp_state_change: sk ffff880038d25480, sk->sk_socket           (null), state (Close)
[   86.737031] [tempesta]   connection error: 192.168.10.1:8080
[   86.737516] [tempesta]   Free msg:           (null)
[   86.737935] [sync_sockets]   Close socket ffff880038d25480 (account=1)
[   86.738496] [sync_sockets]   ss_do_close: sk ffff880038d25480, sk->sk_socket           (null), state (Close)
[   86.739339] [sync_sockets]   ss_tcp_state_change: sk ffff880038d24b80, sk->sk_socket           (null), state (Close)
[   86.740110] [tempesta]   connection error: 192.168.10.1:8080
[   86.740110] [tempesta]   Free msg:           (null)
[   86.740110] [sync_sockets]   Close socket ffff880038d24b80 (account=1)
[   86.740110] [sync_sockets]   ss_do_close: sk ffff880038d24b80, sk->sk_socket           (null), state (Close)
[   86.740110] [sync_sockets]   ss_tcp_state_change: sk ffff880038d24280, sk->sk_socket           (null), state (Close)
[   86.740110] [tempesta]   connection error: 192.168.10.1:8080
[   86.740110] [tempesta]   Free msg:           (null)
[   86.740110] [sync_sockets]   Close socket ffff880038d24280 (account=1)
[   86.740110] [sync_sockets]   ss_do_close: sk ffff880038d24280, sk->sk_socket           (null), state (Close)
[   86.740110] [sync_sockets]   ss_tcp_state_change: sk ffff8800389ef4c0, sk->sk_socket           (null), state (Close)
[   86.740110] [tempesta]   connection error: 192.168.10.1:8080
[   86.740110] [tempesta]   Free msg:           (null)
[   86.740110] [sync_sockets]   Close socket ffff8800389ef4c0 (account=1)
[   86.740110] [sync_sockets]   ss_do_close: sk ffff8800389ef4c0, sk->sk_socket           (null), state (Close)
[   86.768127] [sync_sockets]   ss_tcp_state_change: sk ffff8800389eebc0, sk->sk_socket           (null), state (Close)
[   86.769050] [tempesta]   connection error: 192.168.10.1:8080
[   86.769541] [tempesta]   Free msg:           (null)
[   86.769973] [sync_sockets]   Close socket ffff8800389eebc0 (account=1)
[   86.770539] [sync_sockets]   ss_do_close: sk ffff8800389eebc0, sk->sk_socket           (null), state (Close)
[   86.771399] [sync_sockets]   ss_tcp_state_change: sk ffff8800389ee2c0, sk->sk_socket           (null), state (Close)
[   86.772117] [tempesta]   connection error: 192.168.10.1:8080
[   86.772117] [tempesta]   Free msg:           (null)
[   86.772117] [sync_sockets]   Close socket ffff8800389ee2c0 (account=1)
[   86.772117] [sync_sockets]   ss_do_close: sk ffff8800389ee2c0, sk->sk_socket           (null), state (Close)
[   86.772117] [sync_sockets]   ss_tcp_state_change: sk ffff880038731500, sk->sk_socket           (null), state (Close)
[   86.772117] [tempesta]   connection error: 192.168.10.1:8080
[   86.772117] [tempesta]   Free msg:           (null)
[   86.772117] [sync_sockets]   Close socket ffff880038731500 (account=1)
[   86.772117] [sync_sockets]   ss_do_close: sk ffff880038731500, sk->sk_socket           (null), state (Close)
[   86.772117] [sync_sockets]   ss_tcp_state_change: sk ffff880038730c00, sk->sk_socket           (null), state (Close)
[   86.772117] [tempesta]   connection error: 192.168.10.1:8080
[   86.772117] [tempesta]   Free msg:           (null)
[   86.772117] [sync_sockets]   Close socket ffff880038730c00 (account=1)
[   86.772117] [sync_sockets]   ss_do_close: sk ffff880038730c00, sk->sk_socket           (null), state (Close)
[   86.859337] [sync_sockets]   ss_tcp_state_change: sk ffff880038730300, sk->sk_socket           (null), state (Close)
[   86.860251] [tempesta]   connection error: 192.168.10.1:8080
[   86.860746] [tempesta]   Free msg:           (null)
[   86.861173] [sync_sockets]   Close socket ffff880038730300 (account=1)
[   86.861740] [sync_sockets]   ss_do_close: sk ffff880038730300, sk->sk_socket           (null), state (Close)
[   86.862591] [sync_sockets]   ss_tcp_state_change: sk ffff880038c45540, sk->sk_socket           (null), state (Close)
[   86.863326] [tempesta]   connection error: 192.168.10.1:8080
[   86.863326] [tempesta]   Free msg:           (null)
[   86.863326] [sync_sockets]   Close socket ffff880038c45540 (account=1)
[   86.863326] [sync_sockets]   ss_do_close: sk ffff880038c45540, sk->sk_socket           (null), state (Close)
[   86.863326] [sync_sockets]   ss_tcp_state_change: sk ffff880038c44c40, sk->sk_socket           (null), state (Close)
[   86.863326] [tempesta]   connection error: 192.168.10.1:8080
[   86.863326] [tempesta]   Free msg:           (null)
[   86.863326] [sync_sockets]   Close socket ffff880038c44c40 (account=1)
[   86.863326] [sync_sockets]   ss_do_close: sk ffff880038c44c40, sk->sk_socket           (null), state (Close)
[   86.863326] [sync_sockets]   ss_tcp_state_change: sk ffff880038c44340, sk->sk_socket           (null), state (Close)
[   86.863326] [tempesta]   connection error: 192.168.10.1:8080
[   86.863326] [tempesta]   Free msg:           (null)
[   86.863326] [sync_sockets]   Close socket ffff880038c44340 (account=1)
[   86.863326] [sync_sockets]   ss_do_close: sk ffff880038c44340, sk->sk_socket           (null), state (Close)
[   86.984396] [sync_sockets]   ss_tcp_state_change: sk ffff880038799580, sk->sk_socket           (null), state (Close)
[   86.987166] [tempesta]   connection error: 192.168.10.1:8080
[   86.988368] [tempesta]   Free msg:           (null)
[   86.988368] [sync_sockets]   Close socket ffff880038799580 (account=1)
[   86.988368] [sync_sockets]   ss_do_close: sk ffff880038799580, sk->sk_socket           (null), state (Close)
[   86.988368] [sync_sockets]   ss_tcp_state_change: sk ffff880038798c80, sk->sk_socket           (null), state (Close)
[   86.988368] [tempesta]   connection error: 192.168.10.1:8080
[   86.988368] [tempesta]   Free msg:           (null)
[   86.988368] [sync_sockets]   Close socket ffff880038798c80 (account=1)
[   86.988368] [sync_sockets]   ss_do_close: sk ffff880038798c80, sk->sk_socket           (null), state (Close)
[   86.988368] [sync_sockets]   ss_tcp_state_change: sk ffff880038798380, sk->sk_socket           (null), state (Close)
[   86.988368] [tempesta]   connection error: 192.168.10.1:8080
[   86.988368] [tempesta]   Free msg:           (null)
[   86.988368] [sync_sockets]   Close socket ffff880038798380 (account=1)
[   86.988368] [sync_sockets]   ss_do_close: sk ffff880038798380, sk->sk_socket           (null), state (Close)
[   86.988368] [sync_sockets]   ss_tcp_state_change: sk ffff88003855d5c0, sk->sk_socket           (null), state (Close)
[   86.988368] [tempesta]   connection error: 192.168.10.1:8080
[   86.988368] [tempesta]   Free msg:           (null)
[   86.988368] [sync_sockets]   Close socket ffff88003855d5c0 (account=1)
[   86.988368] [sync_sockets]   ss_do_close: sk ffff88003855d5c0, sk->sk_socket           (null), state (Close)
[   87.240238] [sync_sockets]   ss_tcp_state_change: sk ffff88003855ccc0, sk->sk_socket           (null), state (Close)
[   87.241170] [tempesta]   connection error: 192.168.10.1:8080
[   87.241653] [tempesta]   Free msg:           (null)
[   87.242076] [sync_sockets]   Close socket ffff88003855ccc0 (account=1)
[   87.242633] [sync_sockets]   ss_do_close: sk ffff88003855ccc0, sk->sk_socket           (null), state (Close)
[   87.243476] [sync_sockets]   ss_tcp_state_change: sk ffff88003855c3c0, sk->sk_socket           (null), state (Close)
[   87.244219] [tempesta]   connection error: 192.168.10.1:8080
[   87.244219] [tempesta]   Free msg:           (null)
[   87.244219] [sync_sockets]   Close socket ffff88003855c3c0 (account=1)
[   87.244219] [sync_sockets]   ss_do_close: sk ffff88003855c3c0, sk->sk_socket           (null), state (Close)
[   87.244219] [sync_sockets]   ss_tcp_state_change: sk ffff880038b9b600, sk->sk_socket           (null), state (Close)
[   87.244219] [tempesta]   connection error: 192.168.10.1:8080
[   87.244219] [tempesta]   Free msg:           (null)
[   87.244219] [sync_sockets]   Close socket ffff880038b9b600 (account=1)
[   87.244219] [sync_sockets]   ss_do_close: sk ffff880038b9b600, sk->sk_socket           (null), state (Close)
[   87.244219] [sync_sockets]   ss_tcp_state_change: sk ffff880038b9ad00, sk->sk_socket           (null), state (Close)
[   87.244219] [tempesta]   connection error: 192.168.10.1:8080
[   87.244219] [tempesta]   Free msg:           (null)
[   87.244219] [sync_sockets]   Close socket ffff880038b9ad00 (account=1)
[   87.244219] [sync_sockets]   ss_do_close: sk ffff880038b9ad00, sk->sk_socket           (null), state (Close)
[   87.781385] [sync_sockets]   ss_tcp_state_change: sk ffff880038b9a400, sk->sk_socket           (null), state (Established)
[   87.782403] [tempesta]   connected: 192.168.10.1:8080
[   87.782879] [sync_sockets]   ss_tcp_state_change: sk ffff880038c95640, sk->sk_socket           (null), state (Established)
[   87.783902] [tempesta]   connected: 192.168.10.1:8080
[   87.784379] [sync_sockets]   ss_tcp_state_change: sk ffff880038c94d40, sk->sk_socket           (null), state (Established)
[   87.785367] [tempesta]   connected: 192.168.10.1:8080
[   87.785367] [sync_sockets]   ss_tcp_state_change: sk ffff880038c94440, sk->sk_socket           (null), state (Established)
[   87.785367] [tempesta]   connected: 192.168.10.1:8080
[  387.882987] [sync_sockets]   ss_tcp_state_change: sk ffff880038b9a400, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   Peer connection closing
[  387.886807] [tempesta]   connection lost: 192.168.10.1:8080
[  387.886807] [tempesta]   Free msg:           (null)
[  387.886807] [sync_sockets]   Close socket ffff880038b9a400 (account=1)
[  387.886807] [sync_sockets]   ss_do_close: sk ffff880038b9a400, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   free rcv skb ffff880038a9f1c0
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038c95640, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   Peer connection closing
[  387.886807] [tempesta]   connection lost: 192.168.10.1:8080
[  387.886807] [tempesta]   Free msg:           (null)
[  387.886807] [sync_sockets]   Close socket ffff880038c95640 (account=1)
[  387.886807] [sync_sockets]   ss_do_close: sk ffff880038c95640, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   free rcv skb ffff880038a9f1c0
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038c94d40, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   Peer connection closing
[  387.886807] [tempesta]   connection lost: 192.168.10.1:8080
[  387.886807] [tempesta]   Free msg:           (null)
[  387.886807] [sync_sockets]   Close socket ffff880038c94d40 (account=1)
[  387.886807] [sync_sockets]   ss_do_close: sk ffff880038c94d40, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   free rcv skb ffff880038a9f1c0
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038c94440, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   Peer connection closing
[  387.886807] [tempesta]   connection lost: 192.168.10.1:8080
[  387.886807] [tempesta]   Free msg:           (null)
[  387.886807] [sync_sockets]   Close socket ffff880038c94440 (account=1)
[  387.886807] [sync_sockets]   ss_do_close: sk ffff880038c94440, sk->sk_socket           (null), state (Close Wait)
[  387.886807] [sync_sockets]   free rcv skb ffff880038a9f1c0
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038077680, sk->sk_socket           (null), state (Established)
[  387.886807] [tempesta]   connected: 192.168.10.1:8080
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038076d80, sk->sk_socket           (null), state (Established)
[  387.886807] [tempesta]   connected: 192.168.10.1:8080
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038076480, sk->sk_socket           (null), state (Established)
[  387.886807] [tempesta]   connected: 192.168.10.1:8080
[  387.886807] [sync_sockets]   ss_tcp_state_change: sk ffff880038a296c0, sk->sk_socket           (null), state (Established)
[  387.886807] [tempesta]   connected: 192.168.10.1:8080
```
